### PR TITLE
[ci skip] added .gitattributes

### PR DESCRIPTION
**Pull request by lichunqiang**

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+/tests              export-ignore
+/.gitattributes     export-ignore
+/.gitignore         export-ignore
+/.scrutinizer.yml   export-ignore
+/.travis.yml        export-ignore
+/CHANGELOG.md       export-ignore
+/CONTRIBUTING.md    export-ignore
+/README.md          export-ignore
+/phpunit.xml.dist   export-ignore
```
Comment (cebe) [inline .gitattributes]: CHANGELOG, CONTRIBUTING and README should be included in the export.

Comment (lichunqiang) [inline .gitattributes]: Done~

File: .gitattributes [hunk 2]
```diff
@@ -0,0 +1,8 @@
+* text=auto
+
+/tests              export-ignore
```
Comment (dereuromark) [inline .gitattributes]: Personally, I always like to have the tests along with the code to see how it is used etc.
It doesnt hurt to export IMO.

**Comment by dereuromark** [comment]
The others are fine to be removed (export-ignore) IMO.

**Comment by cebe** [comment]
added a .gitattributes that keeps tests. Thanks @lichunqiang and @dereuromark !

**Pull request closed without merging**